### PR TITLE
Adicionado suporte a branch coverage

### PR DESCRIPTION
**Pull request by velo**

### CODE REVIEW

File: coverage-incrementer [hunk 1]
```diff
@@ -10,7 +10,10 @@ adjust() {
   local project="$(echo "${report/\/target\/coverage-reports\/jacoco.xml/}")"
   local covered="$(ruby "$DIR"/instruction-coverage-extractor.rb -jacoco "$report")"
   local rule="$(ruby "$DIR"/instruction-coverage-extractor.rb -pom "$project/pom.xml")"
-  echo "$project rules={$rule} vs actual={$covered}"
+  if [[ $rule != $covered ]]
```
Comment (caarlos0) [inline coverage-incrementer]: this is old/deprecated bash syntax, use this instead:

``` sh
if [ "$rule" != "$covered" ]; then
  echo "$project rules={$rule} vs actual={$covered}"
fi
```

**Comment by caarlos0** [comment]
Just one comment above.

Thanks!

**Comment by velo** [comment]
done

**Comment by caarlos0** [comment]
Nice, many thanks!

**Pull request merged**